sitecoremaster/Helix-Starter-Kit
# Create solution

## Status
**NEEDS WORK**

## Description
The initial build of a base solution initialization.  

## Todos

## Steps to Test or Reproduce
Ran a few standard tests, need to run more tests of yeoman to confirm all paths work correctly.  Plus confirm that they work with Sitecore installations.
